docs/DeveloperGuide.md

```diff
@@ -853,21 +853,37 @@ Use case ends.
    5. Test case: `searchingredient $20`  
    Expected: Ingredient entries that have the keyword matching `$20` price are listed.
    
-#### F.4. Search for recipe
+#### F.4. Add a recipe
+1. Add a recipe into Kitchen Helper
+    1. Prerequisites: List all the ingredient using the `listingredient all` command.
+    1. Test case: `addrecipe /n warm milk /i HL Milk:1:Dairy`\
+    Expected: Entry can be found using `listingredient all` command. 
+
+#### F.5. Cook a recipe
+1. Cooks the specified recipe and ingredients in the recipe will be automatically deducted.
+    1. Prerequisites: List all the ingredient using the `listingredient all` command.
+    1. Test case (sufficient ingredient): `cookrecipe /n warm milk /p 2`\
+    Expected: A reduction of the ingredients' quantity multiplied by `2` can be noticed when listing the ingredients with `listingredient all`
+    1. Test case (Sufficient even with expired ingredients):  `cookrecipe /n warm milk /p 2`\
+    Expected: The automatic deduction will not be carried out and expired item will be notified to user.
+    1. Test case: (Insufficient even with expired ingredients): `cookrecipe /n warm milk /p 2`\
+    Expected: The automatic deduction will not be carried out.
+
```

Comment:
@alwayshuizhen , Good job adding this. Could you rectify the content page list in the `appendix F` also? Does not correspond when you added all these in.